styles/_variables.scss

```
@@ -1,13 +1,13 @@
 // Variables
 
 // Colors
-$brand_color: #0072ef !default;
+$brand_color: rgba(23, 73, 77, 1) !default;
```

nit: for consistency, let's use the hex values here
```suggestion
$brand_color: #17494d !default;
```